dm/pkg/schema/tracker.go

```diff
@@ -487,36 +483,38 @@ func (tr *Tracker) initDownStreamSQLModeAndParser(tctx *tcontext.Context) error
 	return nil
 }
 
-// getDownStreamTi constructs downstreamTable index cache by tableinfo.
-func getDownStreamTi(ti *model.TableInfo, originTi *model.TableInfo) *downstreamTableInfo {
+// GetDownStreamTi constructs downstreamTable index cache by tableinfo.
+func GetDownStreamTi(ti *model.TableInfo, originTi *model.TableInfo) *DownstreamTableInfo {
 	var (
-		indexCache       *model.IndexInfo
-		availableUKCache = make([]*model.IndexInfo, 0, len(ti.Indices))
-		hasPk            = false
+		absoluteUKIndexInfo  *model.IndexInfo
+		availableUKIndexList = make([]*model.IndexInfo, 0, len(ti.Indices))
```

Comment:
```suggestion
		availableUKIndexList = []*model.IndexInfo{}
```
if we choose to use append below,  not use `make xxx` can redue memory alloc when there is no `availableUKIndex`

Reply:
ok